src/vcpkg/export.cpp

```diff
@@ -106,10 +106,10 @@ namespace vcpkg::Export
             switch (plan_type)
             {
                 case ExportPlanType::ALREADY_BUILT:
-                    print2("The following packages are already built and will be exported:\n", as_string, '\n');
+                    msg::println(msg::format(msgExportingPackages).append_raw("\n" + as_string + "\n"));
```

Comment:
```suggestion
                    msg::println(msg::format(msgExportingPackages).append_raw("\n" + as_string));
```

`println` already adds `\n`. Same below.